### ISSUE TITLE
[a11y] Issue 15: Missing visible text for link (Report Page)

### ISSUE THREAD

**Issue by NewJerseyStyle** [NewJerseyStyle/polis#73]
**Description:**

A link on the report page lacks discernible text, making it inaccessible to screen reader users.

**Steps to Reproduce:**

1. Navigate to the report page.
2. Inspect the link element using a screen reader or developer tools.

**Expected Behavior:**

Links should have visible text that describes their destination.

**Actual Behavior:**

The link lacks visible text.

**Component/Area:** Report Page

**Source:** Auto evaluation

**Affected Elements:** link

**Comment by Natasha1110** [comment]
Could you link the Polis Page in the comments, as I can't seem to find a link to the live view where I can see the link... ?

Also Links/ UX copy can be done in redesign as well

**Comment by Angela1995He** [comment]
@Natasha1110 Here's a report link I found in another comment.
https://pol.is/report/r3bpnywujybyru4rkx92i

**Comment by YumiChen** [comment]
Seems to be the Polis logos at the top and at the bottom of the page( [Axe issue](https://dequeuniversity.com/rules/axe/4.10/link-name?application=AxeChrome)):
![Image](https://github.com/user-attachments/assets/5213b74f-f0b7-43c8-a003-f192a60a4b41)
![Image](https://github.com/user-attachments/assets/60e0829e-2ca1-44c1-ba8f-da124a2b15c8)

Both would navigate user back to [Polis home page](https://dequeuniversity.com/rules/axe/4.10/link-name?application=AxeChrome)
Screen reader would announce that there's a link. Probably we can have some texts to describe the action( `Go to home page` .etc)?

**Comment by NewJerseyStyle** [comment]
Oh, that's a good idea! I didn't even know all those images in report page are links to home page before!

**Comment by YumiChen** [comment]
Ok. I think I'll add the alternative texts( `Go to home page`) for now
